chart/apps/tfl-lab1-prod.yaml

```
@@ -9,11 +9,11 @@ spec:
   source:
     path: 'chart'
     repoURL: 'https://github.com/BaldiSlayer/rofl-lab1'
-    targetRevision: new-deploy
+    targetRevision: adapt-backend-mult-instances
```

```suggestion
    targetRevision: main
```

надо мне собраться с мыслями и починить эту тупую джобу)